### PR TITLE
Remove click listener from experiment name and sha

### LINKED ISSUE #4534: Double click to highlight SHA of table row toggles plot
When I double click on the base commit SHA for a set of experiments in order to copy it (see also #4533), the plots switch is being toggled as a side effect of this.

### PR DESCRIPTION
**Pull request by mattseddon**
Closes #4534

### Demo

https://github.com/iterative/vscode-dvc/assets/37993418/24e0eef0-b3c5-4172-b84d-2f9aa01cae80

### CODE REVIEW

File: webview/src/experiments/components/App.test.tsx [hunk 1]
```diff
@@ -344,56 +346,10 @@ describe('App', () => {
...
-        EXPERIMENT_WORKSPACE_ID
-      )
-
-      testClick(workspace, EXPERIMENT_WORKSPACE_ID)
-      testClick(experimentRunningInWorkspace, 'exp-83425')
-      testClick(screen.getAllByText('main')[1], 'main')
-      testClick(screen.getByText('[exp-e7a67]'), 'exp-e7a67')
-    })
-
-    it('should send a message to the extension to toggle an experiment when Enter or Space is pressed on the row', () => {
```
Comment (mattseddon) [inline webview/src/experiments/components/App.test.tsx:357]: [F] I spent 20-30 minutes trying to fix this test but could not come up with anything that worked. The behaviour does work.

Comment (julieg18) [reply webview/src/experiments/components/App.test.tsx:357]: Maybe I'm misunderstanding this feature, but when I tabbed onto a table row and pressed enter/space nothing was toggled 🤔

Comment (mattseddon) [reply webview/src/experiments/components/App.test.tsx:357]: Focus needs to be on the plot icon. I'll record a demo later

Comment (mattseddon) [reply webview/src/experiments/components/App.test.tsx:352]: https://github.com/iterative/vscode-dvc/assets/37993418/623dd675-0d13-41f1-8eb9-eaff1841f5b1

**Pull request merged**